UsabillaSampleApp/README.md

```diff
@@ -0,0 +1,29 @@
+[![license](https://img.shields.io/badge/license-MIT-brightgreen.svg)](https://github.com/usabilla/usabilla-u4a-react-native/blob/develop/LICENSE)
+
+# Usabilla for Cordova
+
+## Resource Generator
+
+This tool will crop and resize JPEG and PNG source images to generate icons and splash screens for modern iOS, Android, and Windows. `cordova-res` was developed for use with Cordova, but Capacitor and other native runtimes are supported.
+
+### Install
+
+```bash
+$ npm install -g cordova-res
+```
+
+### Usage
+
+To generate resources with all the default options, just run:
+
+```bash
+$ cordova-res
+```
+
+`cordova-res` accepts a platform for the first argument. If specified, resources are generated only for that platform:
```

Comment:
`as first argument` sounds better

Reply:
This is a copied text from the actual cordova-res repository.

Reply:
So we do not want to change it?

Reply:
Done